lib/src/auto_observable.dart

```diff
@@ -1,35 +1,20 @@
-// Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-library observe.src.observable;
+library observe.src.auth_observable;
```

Comment:
type, should this be auto_observable?

BTW you can also remove "library" tags I believe, if it doesn't use a part.